### ISSUE TITLE
Enhanced Property Selector

### ISSUE THREAD

**Issue by workgroupengineering** [AvaloniaUI/Avalonia#10364]
**Is your feature request related to a problem? Please describe.**
Allow Property Selector to comapre target value by less, greater less or equal, greater or equal, and also not equal.

eg:

```xaml
<Window>
   <Window.Styles>
       <Style Selector="Slider[Value]≤20">
            <Setter Property="Background" Value="Red"/>
       </Syle>
       <Style Selector="Slider[Value]≤80">
            <Setter Property="Background" Value="Yellow"/>
       </Syle>
       <Style Selector="Slider[Value]≤80">
            <Setter Property="Background" Value="Green"/>
       </Syle>
  </Window.Styles>
  <Slider Minimum="0" Maximum="100" Value="0"/>
</Window>
```

**Describe the solution you'd like**

**Describe alternatives you've considered**
Behaviors with PseudoClasses can also be used but it is verbose.

**Additional context**

**Comment by pr8x** [comment]
I agree, we have `[Property=Foo]` so we should have `[Property~=Foo]` (unequal) at the very least.

**Comment by maxkatz6** [comment]
CSS supports:
CSS [attribute~="value"] Selector
CSS [attribute|="value"] Selector
CSS [attribute^="value"] Selector
CSS [attribute$="value"] Selector
CSS [attribute*="value"] Selector

https://www.w3schools.com/css/css_attribute_selectors.asp

**Comment by maxkatz6** [comment]
There are also "s" and "i" modifiers.
https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

**Comment by workgroupengineering** [comment]
I've updated the first post. Add your considerations.
